crates/trie/sparse/src/state.rs

```diff
@@ -178,31 +177,35 @@ where
         proof: impl IntoIterator<Item = (Nibbles, Bytes)>,
     ) -> SparseStateTrieResult<()> {
         assert!(!self.retain_updates);
+        // TODO this method doesn't seem to be used, can we remove it?
```

Comment:
I see, yeah it does look unused, I think we can file an issue to remove, wdyt @shekhirin ?